windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,69 @@
+
+[IMPORTANT]
+====
+Users running WMCO 2.0.2 and below must uninstall the operator and reinstall WMCO 3.0.0 to work around a known bug preventing users from upgrading from WMCO 2.0.2 to WMCO 3.0.0. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1983153[*BZ#1983153*])
+====
+
+include::modules/wmco-prerequisites.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+Running Windows container workloads is not supported for clusters in a restricted network or disconnected environment.
+====
+
+Version 3.x of the WMCO is only compatible with {product-title} 4.8.
+
+[id="wmco-3-0-0-improvements"]
+=== New features and improvements
+
+This release adds the following new features and improvements.
+
+// Addresses Github #32620 - https://github.com/openshift/openshift-docs/pull/32620
+==== Clarified limits on custom VXLAN port selection
+
+Users must not select a custom VXLAN port when using the latest version of Windows server.
+
+[id="wmco-3-0-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, the load balancer service would become unstable when the backing deployment had multiple pods scheduled on different Windows nodes. This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[**BZ#1905950**])
+
+* Previously, WMCO added the public key annotation, `windowsmachineconfig.openshift.io/pub-key-hash`, to Linux nodes. Now, WMCO no longer adds an annotation to Linux nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930791[**BZ#1930791**])
+
+* Previously, when users provided an invalid private key, the Windows Machine Config Operator (WMCO) would fail. With this update, the WMCO produces an error alerting the user of an invalid key. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929579[*BZ#1929579*])
+
+* Previously, the kube-proxy service would crash upon the creation of a load balancer service when the backing development had multiple pods scheduled on different Windows nodes. This issue has now been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[**BZ#1939968**])
+
+* Previously, the `windows_exporter` component reported various metrics as `windows_*`. This error caused some of the node-level metrics that provide insight into the nodes through telemetry services to go unreported. Now, the componets export correctly showing all of the expected metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1948037[**BZ#1948037**])
+
+=== RHSA-2021:76465-04 - Windows Container support for {product-title} security update
+
+As part of the previously noted bug fix (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946538[**BZ#1946538**]), an update for Windows kube-proxy is now available for Red Hat Windows Machine Config Operator 2.0.1. Details of the update are documented in the link:https://access.redhat.com/errata/RHSA-2021:2130[RHSA-2021:2130] advisory.
```

Comment:
I'm a bit confused about this v2 content's inclusion, and the reference to a v2 advisory under a v3 advisory header.

Reply:
@aravindhp Should we keep this included?

Reply:
I will defer to whatever pattern the docs team has been following regarding maintaining advisories across versions.

Reply:
I think the main thing to consider is: Is this relevant to v3 users? If not, I would lean towards removing it. v2 users should be getting relevant info in the v2 Rel Notes. But I don't really know enough about WinC to make an educated statement on this case :)

Reply:
This is not relevant to v3 users given it was fixed in the v2 timeframe.